### PR TITLE
Feature/direct transfer

### PR DESCRIPTION
**Pull request by woutslakhorst**
This PR adds an extra button to assign a single organization for the transfer. It cancels the other organizations, handles auth credentials and sends notifications!

### CODE REVIEW

**Review by dmeijboom — CHANGES_REQUESTED** [review]

File: domain/transfer/sender_service.go [hunk 1]
```diff
@@ -121,20 +107,216 @@ func (s service) CreateNegotiation(ctx context.Context, customerID int, transfer
...
+	transfer, err := s.transferRepo.FindByID(ctx, customerID, transferID)
+	if err != nil {
+		return nil, err
+	}
+	if transfer == nil {
+		return nil, fmt.Errorf("transfer with ID: %s, not found", transferID)
+	}
+
+	var (
+		negotiation   *domain.TransferNegotiation
```
Comment (stevenvegt) [inline domain/transfer/sender_service.go:130]: This value is not set but used on line 149

File: api/transfer.go [hunk 2]
```diff
@@ -125,54 +125,75 @@ func (w Wrapper) StartTransferNegotiation(ctx echo.Context, transferID string) e
...
+//		// Update transfer.Status = assigned
+//		transfer.Status = domain.TransferStatusAssigned
+//		return &transfer, nil
+//	})
+//	if err != nil {
+//		return err
+//	}
+//	return ctx.JSON(http.StatusOK, negotiation)
+//}
+
```
[the remark this answers is not in the document]
Comment (woutslakhorst) [reply api/transfer.go:196]: I was hoping someone else could answer that question. I'll remove it, since n=2 don't know if it's used.

**Review by stevenvegt — CHANGES_REQUESTED** [review]
I get a nil dereference error on `domain/transfer/sender_service.go #149`. To reproduce:
* New transfer
* Request care org A
* Assign to care org B

File: domain/transfer/sender_service.go (hunk 1 above)
Comment (woutslakhorst) [reply domain/transfer/sender_service.go:130]: simple fix:

Screenshot 2021-09-08 at 10 33 49

**Review by dmeijboom — APPROVED** [review]
LGTM

File: nuts/client/auth/generated.go [hunk 3]
```diff
@@ -1,6 +1,6 @@
 // Package auth provides primitives to interact with the openapi HTTP API.
 //
-// Code generated by github.com/deepmap/oapi-codegen version v1.8.2 DO NOT EDIT.
+// Code generated by github.com/deepmap/oapi-codegen version v1.7.1 DO NOT EDIT.
```
Comment (stevenvegt) [inline nuts/client/auth/generated.go:3]: go install github.com/deepmap/oapi-codegen/cmd/oapi-codegen@v1.8.2

**Review by stevenvegt — APPROVED** [review]

**Pull request merged**